src/Carbon/Lang/nl.php

```diff
@@ -71,9 +71,9 @@
     'period_recurrences' => ':count keer',
     'period_interval' => function ($interval) {
```

Comment:
Actually we could enforce `string` here. In the Carbon context, it will always receive a string:
```suggestion
    'period_interval' => function (string $interval) {
```

Reply:
`TypeError : Carbon\Translator::{closure}(): Argument #1 ($interval) must be of type string, null given, called in src/Carbon/Translator.php on line 168`

The tests pass when you replace `null` with `''` in this code: https://github.com/briannesbitt/Carbon/blob/27a1ff3b2daf97a1c9d033fe0b1233abbbb8930a/src/Carbon/Translator.php#L166-L169

But it might be a breaking change for people using this translator (or is it only used internally?)

Reply:
The only point where we use `period_interval` translation is:
```php
$this->translate('period_interval', [':interval' => $this->dateInterval->forHumans([
    'join' => true,
])], null, $translator)
```
So it's only breaking for people using:
```php
->translate('period_interval', [':interval' => null])
```
But indeed it could be smoother to just cast it. Then I guess we could cast only once at the beginning.

Reply:
`CarbonInterval::forHumans()` always returns a string. Please see my latest commit (0a396b5), the problem was caused by the test that checks whether translation key exists and passes no arguments. No need to cast, I've fixed the test.